Makefile

```diff
@@ -81,3 +78,47 @@ ui-test:
 
 ui-audit:
 	npm audit
+
+# Directory for coverage data
+coverage:
```

Comment:
I think having the `coverage` target do nothing other than create the dir will be confusing. `make coverage` seems like a high-level user command.

Reply:
Yeah I thought about that. Mixing "classic" file targets and jobs isn't great. I will see if I can make that clearer or get rid of the target.